tests/test_trace.py

```diff
@@ -511,6 +520,27 @@ def test_send_big_trace(wrapped_post):
         headers={'Authorization': 'Bearer {}'.format(trace.token)}
     )
 
```

Comment:
let's add a test for this specific case - 
have a key to ignore in return value, and validate it's actually removed